mods/ra/maps/soviet-06a/soviet06a.lua

```diff
@@ -87,9 +87,11 @@ Trigger.OnKilled(Apwr2, function(building)
 	BaseApwr2.exists = false
 end)
 
-Trigger.OnKilled(Dome, function()
-	player.MarkCompletedObjective(sovietObjective2)
-	Media.PlaySpeechNotification(player, "ObjectiveMet")
+Trigger.OnKilledOrCaptured(Dome, function()
+	Trigger.AfterDelay(DateTime.Seconds(2), function ()
```

Comment:
No space between `function` and the parentheses, please.

Reply:
Oops.
Updated.